handlers/handler-loadbalance/src/main/java/io/servicecomb/loadbalance/LoadBalanceRetryHandler.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2017 Huawei Technologies Co., Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.servicecomb.loadbalance;
+
+import java.net.ConnectException;
+import java.net.SocketException;
+import java.net.SocketTimeoutException;
+import java.util.List;
+
+import com.google.common.collect.Lists;
+import com.netflix.client.RetryHandler;
+import com.netflix.client.Utils;
+
+public class LoadBalanceRetryHandler implements RetryHandler {
+
+  private final int retrySameServer;
+  private final int retryNextServer;
+  private final boolean retryEnabled;
+
+  private List<Class<? extends Throwable>> retryOnSameExceptions = Lists
+      .newArrayList(new Class[]{SocketTimeoutException.class, ConnectException.class});
```

Comment:
not complete
because transports will not thow these exceptions.
to support this, we should modify transport is this issue or create another issue to trace this problem.

Reply:
To init these exceptions is in order to be compatible with original implement.
In my opinion, we could create another issue to trace transport module problem and do more test.

Reply:
We should make what we have implemented work, even if it's a bug of old system, or we can't judge if your implementation is correct without a running example.